README.md

```diff
@@ -0,0 +1,127 @@
+# Openshift APIs for Data Protection (OADP) Python Client Wrapper 
+
+Manage Openshift adp Custom Resources 
+
+Read more about OADP
+
+https://access.redhat.com/articles/5456281
+
+
+## Installation
+
+
+```bash
+# Install from repo
+git clone https://github.com/oadp-qe/openshift-adp-python-wrapper
+pip install <path to>/openshift-adp-python-wrapper
+```
+
+## Usage Example
+### Creating a dpa CR, wait for a successful reconciliation and delete it. 
+note: 
+1. The j2 helper module includes a build-in filter for getting env variable values: `env_override`
+2. NO Templates are not provided with this project.Please refer to OADP Documentation for more information. 
+
+openshift_resource/oadp/data_protection_application
+```python
+# openshift_resource/oadp/data_protection_application
+r = constants.resources
+class DataProtectionApplication(NamespacedResource):
+    api_group = r.OADP_API_GROUP
+
+    def wait_for_successful_reconciliation(self):
+        self.wait_for_condition(condition=r.CONDITION_RECONCILED, status=r.STATUS_SUC_RECONCILED)
+
+    def wait_for_failed_reconciliation(self):
+        self.wait_for_condition(condition=r.CONDITION_RECONCILED, status=r.STATUS_FAILED_TO_RECONCILE)
+```
+templates/awsdpa.j2
+```yaml
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  namespace: {{ namespace }}
+  name: {{ name }}
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+      - openshift
+      - azure
+{% if enable_csi is defined and enable_csi %}
+      - csi
+{% endif %}
+{% if enable_restic is defined  and enable_restic %}
+    restic:
+      enable:  {{ enable_restic  }}
+{% endif }} 
+  backupLocations:
+    - name: default
+      velero:
+        provider: azure
```

Comment:
I think it's better to just load the provider from somewhere so it would be agnostic and generic, without referring some specific provider configuration

Reply:
We use different templates for the different providers